src/UI/templates/js/Core/ui.js

```diff
@@ -0,0 +1,51 @@
+var il = il || {};
+il.UI = il.UI || {};
+
+(function($, UI) {
+
+	UI.core = (function ($) {
```

Comment:
If the above approach is used, here we would have something like 

`UI.core.replace = (function ($) {...`